src/Components/NavBar/NavBar.js

```diff
@@ -1,58 +1,133 @@
 import React from "react";
 import "./NavBar.css";
+import hamburger from "../../images/hamburger.png";
+import exit from "../../images/exit.png";
 import { Link } from "react-router-dom";
-
 import patreon from "../../images/Patreon.png";
-import search from "../../images/search.png";
+import hugeLogo from "../../images/hugeLogo.png";
+
+function hamburgerClick() {
+    var hamburgerLinks = document.getElementById("hamburgerLinks");
+    var hamburgericon = document.getElementById("hamburgericon");
+
+    hamburgerLinks.classList.toggle("hidden");
+
+    if (hamburgericon.src === hamburger) {
+        hamburgericon.src = exit;
+    } else {
+        hamburgericon.src = hamburger;
+    }
+}
 
 export default function NavBar() {
     return (
         <div className="navContainer">
-            <ul className="navLinks">
-                <li className="navLinkOuter">
-                    <Link className="navLink" to="/">
-                        Home
+            <div className="navMainContainer">
+                <ul className="navLinks">
+                    <li className="navLinkOuter">
```

Comment:
you could minify this by having an array of strings and then just loop over them here and make a
`
<li className="navLinkOuter">
    <link className="navLink" to="/{string_from_array}">
        {string_from_array}
    </link>
</li>
`
something like that maybe?